crates/secrets/src/lib.rs

```diff
@@ -14,6 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#[cfg(feature = "aws-secret-store")]
+pub mod aws;
```

Comment:
Let's make this specific. `aws_secret_store`

Reply:
Updated to `aws_secrets_manager` (file name, struct name, feature flag so we have consistent naming everywhere). Thank you @lukekim!